functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/execution/CommandOnMultipleNodesSpec.groovy

```diff
@@ -52,7 +52,7 @@ class CommandOnMultipleNodesSpec extends SeleniumBase{
         commandPage.commandTextField.sendKeys "echo running test '" + this.class.name.toString() + "'"
         commandPage.runButton.click()
         def href = commandPage.runningButtonLink().getAttribute("href")
-        commandPage.driver.get href + "#output"
+        commandPage.driver.get href + "#outputL3"
```

Comment:
From @smartinellibenedetti 
> the section being tested actually has the virtual scroll component - it reuses the dom elements whenever it can, so by adding the line parameter to the url, will force the page to scroll and get the dom the latest state.